docs/designers-developers/designers/user-interface.md

```diff
@@ -41,9 +41,11 @@ Please note that selection and focus can be different. An image block can be sel
 ![Settings Sidebar](https://cldup.com/iAqrn6Gc8o-3000x3000.png)
```

Comment:
A new screenshot is needed to correspond with these changes to the text as well otherwise it'll be confusing to have differing names here. Not quite sure who might able to make this happen.

Reply:
I found the Figma file and made a duplicate. Then edited the screen.

![04-Sidebar](https://user-images.githubusercontent.com/5323259/104078434-4a102280-521e-11eb-8389-7ab1647d9478.png)

I am thinking this is Mark who have made these screens.
@mapk 

Let's get some feedback from him on these adjustments.

Reply:
Looks good to me fwiw!

Reply:
I have not heard anything from Mark, so I say let's go for it..:)